packages/components/psammead-numbered-list/src/index.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import { string, oneOf, shape } from 'prop-types';
+import { getBodyCopy } from '@bbc/gel-foundations/typography';
```

Comment:
Can you list the dependencies in the `dependencies` or `peerDependencies` sections of `package.json`